README.md

```diff
@@ -164,11 +167,32 @@ The secret can be created by executing the following command:
 oc create secret generic github-group-sync --from-literal=token=<token>
 ```
 
+##### As a GitHub app
+
+When authenticating as a Github App, the following keys are required:
+* `privateKey` and `integrationId`
+
+
+###### First create a github app
+
+In GitHub, go to developer-settings -> github apps. Create a new app, it does not need webhook callbacks.
+Generate a private-key and download it, under "permissions and events" the app will need read-only access to the "Members" permission in the "Organization" section.
+Take note of the "App ID" as you need it for later.
```

Comment:
Notice how "App ID" is used here, but gets added to the secret as an `integrationID`